.circleci/config.yml

```diff
@@ -27,6 +27,7 @@ jobs:
           command: |
             python -m venv ~/venv
             . ~/venv/bin/activate
+            pip install --upgrade pip
```

Comment:
@glatard I had to add this to circleci/config.yml so that it uses the latest version of pip instead of an old one. Do you see any objections to that?

The build seems to pass now... I can also issue a separate PR just for that. Let me know.

Reply:
looks very relevant to me!